src/Tickets/Site_Health/Subsections/Features/Tickets_Commerce_Subsection.php

```diff
@@ -0,0 +1,262 @@
+					'event-tickets'
+				),
+				'value'    => $this->get_tickets_commerce_thousands_separator(),
+				'priority' => 320,
+			],
+			[
+				'id'       => 'tickets_commerce_number_of_decimals',
+				'title'    => esc_html__(
+					'Tickets Commerce Number of Decimals',
+					'event-tickets'
+				),
+				'value'    => $this->get_tickets_commerce_number_of_decimals(),
+				'priority' => 330,
+			],
+		];
+	}
+
+	/**
+	 * Calculates the average order total for tickets commerce.
+	 *
+	 * @return string Formatted average price.
+	 * */
+	private function get_tickets_commerce_average_order_total(): string {
+		$tickets_commerce_ticket_prices = tribe( Tickets_Repository::class )->per_page( -1 )->pluck( 'price' );
+		$total                          = 0;
+		$count                          = 0;
+
+		foreach ( $tickets_commerce_ticket_prices as $price ) {
+			if ( $price === 'Free' || $price === '' || $price === null ) {
+				// Skip free or empty prices for average calculation.
+				continue;
+			} else {
+				// Match the number with international currency format.
+				preg_match(
+					'/\d+([,.]\d+)?/',
+					$price,
+					$matches
+				);
+				if ( isset( $matches[0] ) ) {
+					// Convert to a standard number format (replace comma with period).
+					$number = floatval(
+						str_replace(
+							',',
+							'.',
+							$matches[0]
+						)
+					);
+					$total  += $number;
+
+					++$count;
+				}
+			}
+		}
+
+		// Calculate the average price, avoid division by zero.
+		$tickets_commerce_average_price = $count > 0 ? $total / $count : 0;
+
+		// Format the average price with two decimal points.
+		$tickets_commerce_formatted_average_price = number_format(
+			$tickets_commerce_average_price,
+			2,
+			'.',
+			','
+		);
+
+		return $tickets_commerce_formatted_average_price;
+	}
+
+	/**
+	 * Determines if Tickets Commerce is in test mode.
+	 *
+	 * @return string 'True' if Tickets Commerce is in test mode, 'False' otherwise.
+	 */
+	private function is_tickets_commerce_test_mode(): string {
+		return $this->get_boolean_string(
+			tribe_get_option(
+				'tickets-commerce-test-mode',
+				false
+			)
+		);
+	}
+
+	/**
+	 * Checks if Stripe is connected with Tickets Commerce.
+	 *
+	 * @return string 'True' if Stripe is connected, 'False' otherwise.
+	 */
+	private function is_tickets_commerce_stripe_connected(): string {
+		return tribe( Stripe_Gateway::class )->is_enabled() ? 'True' : 'False';
+	}
+
+	/**
+	 * Determines if PayPal is connected with Tickets Commerce.
+	 *
+	 * @return string 'True' if PayPal is connected, 'False' otherwise.
+	 */
+	private function is_tickets_commerce_paypal_connected(): string {
+		return tribe( PayPal_Gateway::class )->is_enabled() ? 'True' : 'False';
+	}
+
+	/**
+	 * Retrieves the currency code set in Tickets Commerce.
+	 *
+	 * @return string The currency code.
+	 */
+	private function get_tickets_commerce_currency_code(): string {
+		return tribe_get_option(
+			'tickets-commerce-currency-code'
+		);
```

Comment:
Should we be including the fallback/defaults for all these options?
```suggestion
		return tribe_get_option(
			'tickets-commerce-currency-code',
			'USD'
		);
```
Reference: `TEC\Tickets\Commerce\Utils\Currency::$currency_code_fallback`